src/bcs/SchottkyEmissionBC.C

```diff
@@ -104,8 +104,8 @@ SchottkyEmissionBC::SchottkyEmissionBC(const InputParameters & parameters)
 ADReal
 SchottkyEmissionBC::computeQpResidual()
 {
-  _v_thermal = std::sqrt(8 * ZAPDOS_CONSTANTS::e * 2.0 / 3 * std::exp(_mean_en[_qp] - _u[_qp]) /
-                         (libMesh::pi * _massem[_qp]));
+  _v_thermal = sqrt(8 * ZAPDOS_CONSTANTS::e * 2.0 / 3 * exp(_mean_en[_qp] - _u[_qp]) /
```

Comment:
```suggestion
  using std::sqrt;
  using std::exp;
  _v_thermal = sqrt(8 * ZAPDOS_CONSTANTS::e * 2.0 / 3 * exp(_mean_en[_qp] - _u[_qp]) /
```

For clarity, would prefer to have these prior to first use.

Reply:
damn, script fail